CCT_cold/config.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": “Columbia Card Task Cold Version“,
+        "run": [
```

Comment:
See the red here? The " character you are using is not the right one (what are you using for a text editor?) Try copy pasting from a file that you know does not have this invalid character.